internal/venice-common/src/main/java/com/linkedin/venice/offsets/OffsetRecord.java

```diff
@@ -346,10 +336,6 @@ public String toSimplifiedString() {
   }
 
   private String getPartitionUpstreamOffsetString() {
```

Comment:
Can we delete this method? when we concat map with string it should call toString on the map internally